src/components/HomeHeader.js

```diff
@@ -81,8 +87,14 @@ class HomeHeader extends React.Component {
   }
 }
 
+const mapStateToProps = (state) => {
+  return {
+    analytics: state.charts.analytics
+  }
+};
+
 const mapDispatchToProps = {
-  filterChartsOnSearch
```

Comment:
Is this not required?

Reply:
Does not really break anything. Will put it back in.